### PR TITLE
Enable androidx and swift support plugin in example

### PR DESCRIPTION
**Pull request by vegaro**
With `"cordova-android": "^9.0.0"` there is a new preference `AndroidXEnabled` that enables AndroidX. I also added the `cordova-plugin-add-swift-support` plugin because I was getting the `error: Value for SWIFT_VERSION cannot be empty.` error when building the project

### CODE REVIEW

File: examples/cordova-sample/MyApp/config.xml [hunk 1]
```diff
@@ -23,4 +23,5 @@
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
     </platform>
+    <preference name="AndroidXEnabled" value="true" />
```
Comment (aboedo) [inline examples/cordova-sample/MyApp/config.xml:26]: finally 🙌

File: examples/cordova-sample/MyApp/package.json [hunk 2]
```diff
@@ -23,15 +23,15 @@
     "plugins": {
       "cordova-plugin-whitelist": {},
       "cordova-annotated-plugin-android": {},
-      "cordova-plugin-purchases": {}
+      "cordova-plugin-add-swift-support": {}
```
Comment (aboedo) [inline examples/cordova-sample/MyApp/package.json:26]: shouldn't `purchases` be here?

File: examples/cordova-sample/MyApp/package.json [hunk 3]
```diff
@@ -23,15 +23,15 @@
...
     },
     "platforms": [
-      "ios",
```
Comment (aboedo) [inline examples/cordova-sample/MyApp/package.json:29]: was this intentional?

Comment (vegaro) [reply examples/cordova-sample/MyApp/package.json:29]: no lol, sorry I think I pushed some stuff that I shouldn't have 🤕

**Review by aboedo — APPROVED** [review]

**Pull request merged**